src/Plugin.php

```diff
@@ -175,13 +176,16 @@ public function onDependenciesChangedEvent()
             $installPathUpdated = $this->updateInstalledPaths();
 
             if ($installPathCleaned === true || $installPathUpdated === true) {
-                $this->saveInstalledPaths();
+                $exitCode = $this->saveInstalledPaths();
             } elseif ($isVerbose) {
                 $io->write(sprintf('<info>%s</info>', self::MESSAGE_NOTHING_TO_INSTALL));
             }
         } elseif ($isVerbose) {
             $io->write(sprintf('<info>%s</info>', self::MESSAGE_NOT_INSTALLED));
+            $exitCode = 1;
```

Comment:
The exit code is only set if we are running in verbose mode.
So running update/install works fine until we are running in verbose mode, then it breaks.

i propose that with and without verbose mode the exitCode should be the same, verbose is just an log/output level modifier and  should not change behaviour

Reply:
Agree. I've implemented your suggestion.

Reply:
Good point and thanks for fixing that.